ros2service/ros2service/verb/echo.py

```diff
@@ -0,0 +1,236 @@
+                 "length > '--truncate-length', by default they are truncated "
+                 "after '--truncate-length' elements with '...''")
+        parser.add_argument(
+            '--truncate-length', '-l', type=unsigned_int, default=DEFAULT_TRUNCATE_LENGTH,
+            help='The length to truncate arrays, bytes, and string to '
+                 '(default: %d)' % DEFAULT_TRUNCATE_LENGTH)
+        parser.add_argument(
+            '--no-arr', action='store_true', help="Don't print array fields of messages")
+        parser.add_argument(
+            '--no-str', action='store_true', help="Don't print string fields of messages")
+        parser.add_argument(
+            '--csv', action='store_true',
+            help=(
+                'Output all recursive fields separated by commas (e.g. for '
+                'plotting). '
+                'If --include-message-info is also passed, the following fields are prepended: '
+                'source_timestamp, received_timestamp, publication_sequence_number,'
+                ' reception_sequence_number.'
+            )
+        )
+        parser.add_argument(
+            '--include-message-info', '-i', action='store_true',
+            help='Shows the associated message info.')
+        parser.add_argument(
+            '--client', action='store_true', help="Echo only request sent or response received by service client")
+        parser.add_argument(
+            '--server', action='store_true', help="Echo only request received or response sent by service server")
+
+    def main(self, *, args):
+        self.truncate_length = args.truncate_length if not args.full_length else None
+        self.no_arr = args.no_arr
+        self.no_str = args.no_str
+        self.csv = args.csv
+        self.include_message_info = args.include_message_info
+        self.client = args.client
+        self.server = args.server
+
+        if args.service_type is None:
+            with NodeStrategy(args) as node:
+                self.srv_module = get_service_class(
+                    node, args.service_name, blocking=False, include_hidden_services=True)
+        else:
+            try:
+                self.srv_module = get_service(args.service_type)
+            except (AttributeError, ModuleNotFoundError, ValueError):
+                raise RuntimeError("The service type '%s' is invalid" % args.service_type)
+
+        if self.srv_module is None:
+            raise RuntimeError(
+                'Could not load the type for the passed service')
+
+        event_topic_name = args.service_name + "/_service_event"
+
+        with NodeStrategy(args) as node:
+            self.subscribe_and_spin(
+                node,
+                event_topic_name,
+                self.event_msg_type
+            )
+
+    def subscribe_and_spin(
+            self,
+            node: Node,
+            event_topic_name: str,
+            event_msg_type: MsgType,
+    ) -> Optional[str]:
```

Comment:
It doesn't look like this function is returning anything.
```suggestion
    ) -> None:
```